_tests/integration/helper/validation.go

```diff
@@ -35,14 +33,14 @@ func ValidateConfigExpectation(t *testing.T, ID, expected, actual string, versio
 		fmt.Println("Expected: ", expPth)
 		fmt.Println("Actual: ", actPth)
 
-		_, err = exec.LookPath("opendiff")
-		if err == nil {
-			require.NoError(t, exec.Command("opendiff", expPth, actPth).Start())
-			t.FailNow()
-			return
-		}
-		log.Warnf("opendiff not installed, unable to open config diff")
+		//_, err = exec.LookPath("opendiff")
+		//if err == nil {
```

Comment:
I saw a few commented out code in this PR. Could you remove the unnecessary ones?

Reply:
Sure